### ISSUE TITLE
Make "flip on short edge" an option when printing

### ISSUE THREAD

**Issue by vendi95** [luksan47/mars#125]
When two-sided printing is selected make it possible for the user to choose short edge flipping.  Long edge should stay the default. Also, some help text may be useful.

https://github.com/luksan47/mars/blob/6579899224ab758e4c0ba5d56d271a27a1f9d991/app/Http/Controllers/PrintController.php#L136-L139

**Comment by husar** [comment]
Hello, can you assign it on me? I can solve this issue

**Comment by luksan47** [comment]
Sure, thanks for the help!

**Comment by husar** [comment]
https://github.com/husar/mars/pull/1
check it please

**Comment by luksan47** [comment]
It would be also nice to have a rotation option, would be better to test it only once with the printer.
I've been trying to play with this one, but didn't manage to get it work.
